CONTRIBUTING.md

```diff
@@ -34,10 +32,223 @@ This will automatically link your commit to the GitHub issue, and automatically
 
+We use GitHub Actions for Continuous Integration. As part of this, we have a workflow for releasing the package. This workflow is triggered manually and includes steps for checking out the code, setting up Node.js, installing dependencies, and running the release script.
+To trigger the release workflow, go to the "Actions" tab in the GitHub repository, select the "Release" workflow, and click on "Run workflow".
+
+GitHub Actions for this repo are configured in [./.github/workflows](./.github/workflows).
+
+- [conventional-commits.yml](./.github/workflows/.yml) - checks the title of pull requests to ensure they follow a specified format. It is triggered when a pull request is opened, reopened, edited, or synchronized.
+- [e2e-tests.yml](./.github/workflows/.yml) - runs end-to-end tests for the project. It is triggered when a pull request is opened, reopened, edited, or synchronized.
+- [lint.yml](./.github/workflows/.yml) - checks the code for linting errors. It is triggered when a pull request is opened, reopened, edited, or synchronized.
+- [mitm.yml](./.github/workflows/.yml) - sets up a Man-in-the-Middle (MITM) proxy for testing purposes. It is triggered when a pull request is opened, reopened, edited, or synchronized.
+- [prepare-release.yml](./.github/workflows/prepare-release.yml) - prepares a release by creating a pull request and a GitHub release. It is triggered manually through a workflow dispatch event. The user needs to specify the next SemVer version as an input when triggering the workflow.
+- [prettier.yml](./.github/workflows/.yml) - checks the formatting of the code using Prettier. It is triggered when a pull request is opened, reopened, edited, or synchronized.
+- [publish.yml](./.github/workflows/.yml) - used to publish and release a new version of the project. It is triggered when a pull request is closed and merged into the main branch, and the pull request's head branch starts with `release/`.
+- [size-limit.yml](./.github/workflows/.yml) - uses the `andresz1/size-limit-action` action to calculate the size of the project.
+- [unit-tests.yml](./.github/workflows/.yml) - runs unit tests for the project. It is triggered when a pull request is opened, reopened, edited, or synchronized.
+
 ## Reviewing
 
 A member of the team will review your changes. Once the member has reviewed your changes, they will comment on your pull request. If the member has any questions, they will add a comment to your pull request. If the member is happy with your changes, they will merge your pull request.
+
+## Master Branch Conventions and Mergify
+
+All commits in the master branch should come from squashed GitHub Pull Requests, and those commit messages should follow the [conventionalcommits.org](https://conventionalcommits.org) syntax.
+
+Mergify can take care of enforcing all of this. Just add the `automerge-squash` label to each Pull Request that Mergify should merge. This policy is configured via [./.mergify.yml](./.mergify).
```

Comment:
Mergify has never worked on this repo so far. We've used the `auto-merge (squash)` feature

(I realize this was in the Contributing guide, but it was copied from SDK and this is my first time noticing

Reply:
missed that, thanks!